.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Before I merge: does it make sense to do this as a matrix (is this what you were saying before?)

I do think it may actually be better / more clear to have this *not* be a matrix since it's only ever going to be a single version for linting, but I also don't know if there is a cost (or real benefit).

Reply:
I'm MERGING ANYWAY (and we can tweak this if we decide it should be later)

Reply:
It doesn't really make sense, but after adding the matrix to the build in the first place it was easier to remove a version from the list than to remove the matrix boilerplate, since it's otherwise consistent with the test workflow in every other regard. There's a strong argument for removing it under the Strunk and White rule, it's there for a foolish consistency.

To be honest, it feels like overkill for the project to test on more than one version of Node. It does prevent regressions for 13, but at a certain point you want to encourage people to upgrade since 13 is nowhere on Node's maintenance schedule and 16 will be there until 2024. With Docker, we can ship a specific Node version.